autoload/lsp/ui/vim/output.vim

```diff
@@ -121,16 +115,19 @@ function! lsp#ui#vim#output#floatingpreview(data) abort
 
         let l:opts = s:get_float_positioning(l:height, l:width)
 
-        let s:winid = nvim_open_win(l:buf, v:true, l:opts)
+        let s:winid = nvim_open_win(l:buf, v:false, l:opts)
```

Comment:
I don't know the reason but this and L159 changes make exception sometimes... Hm...

Reply:
My repro steps are the below.


0. Add `let g:lsp_diagnostics_float_cursor = 1`
1. Open some `go` file
2. Edit `fmt.Println()` (this should be marked as error by diagnostics)
3. Put cursor on `P` and wait for display floating diagnostic window.
4. Do `LspHover`.

Reply:
You mean this - (GIF below)
![gifcast_201113113115](https://user-images.githubusercontent.com/56714626/99034759-efeb3c00-25a3-11eb-8424-2fa009f2d90a.gif)

<br>

Seems OK to me...
Sorry if I did some step wrong.. I have never coded Go

Reply:
Are you using Neovim nightly?

Reply:
I took screenshot please check it.

![Kapture 2020-11-13 at 15 17 16](https://user-images.githubusercontent.com/629908/99035604-5d5a9500-25c3-11eb-9453-1d61475284d5.gif)

Reply:
Yes, I'm using neovim nightly.

Reply:
I tried to fix it in the latest commit. Please check.

Reply:
Thank you! It works perfectly.